### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by sulaimangari**
added license for legal purpose

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 sulaimangari
```
Comment (defunctzombie) [inline LICENSE:3]: Its not really appropriate to assign your own copyright to someone else's project.

**Comment by defunctzombie** [comment]
Thanks - I'm not looking to add a license to this repo.

File: LICENSE (hunk 1 above)
Comment (sulaimangari) [reply LICENSE:3]: my bad, I forgot to change it so the license refer to you

**Comment by debloper** [comment]
> I'm not looking to add a license to this repo

As someone who had to be [schooled](https://gist.github.com/chirag64/7853413?permalink_comment_id=3737331#gistcomment-3737331) on the utility of needing to add a License for a public piece of code, I'd strongly recommend that you reconsider.

As someone who despises these parasitic bureaucratic bullshit overheads of IP rights/legalities that plagues free flow of information, without adding functional value to the solution; may I suggest you to at least use something like [Unlicense](https://en.wikipedia.org/wiki/Unlicense), [WTFPL](https://en.wikipedia.org/wiki/WTFPL) etc. if you really want to make your point (about not wanting to add a license) clear, without posing/passing the problem on people who may want to build on top of it.

I am saying this, cause I intend to create a distro using this template; and unless the terms of license is clear from your end (within a reasonable time... let's say a week from the posting of this comment), I will go ahead with a license of my choice. This message is to ensure avoidance of potential contention (if any/ever).

**Comment by defunctzombie** [comment]
The readme clearly states: https://github.com/defunctzombie/jetson-nano-image-maker?tab=readme-ov-file#license

Make of that what you will.

**Comment by debloper** [comment]
That seems sufficient (to me), thanks for the direction for where to find it.

However, this won't be picked up in any [automated check](https://github.com/defunctzombie/jetson-nano-image-maker/community) where folks will be looking for this kind of information. This heavily relies on human effort of reading through every last word, which is very idealistic expectation; but that's just my subjective opinion/feedback.

Thanks for setting up the great automation & sharing with the world.

**Pull request closed without merging**